arangod/Graph/KShortestPathsFinder.cpp

```diff
@@ -155,9 +155,10 @@ bool KShortestPathsFinder::advanceFrontier(Ball& source, Ball const& target,
                                            std::unordered_set<Edge> const& forbiddenEdges,
                                            VertexRef& join) {
   VertexRef vr;
-  FoundVertex* v;
+  FoundVertex *v, *w;
 
   bool success = source._frontier.popMinimal(vr, v);
+  TRI_ASSERT(vr == v->_vertex);
```

Comment:
please:
TRI_ASSERT(v != nullptr);
to avoid udef that we do not see. Before accessing v->_vertex

Reply:
Done.